README.md

```diff
@@ -110,39 +97,13 @@ Follow the same process of deduplication, reinstallation, and re-deduplication a
 
 #### Update ember-cli-typescript
 
-***Note:* Because ember-cli-typescript is part of the build pipeline, the process for updating it differs slightly between apps and addons!**
-
-##### In apps
-
-In apps, you can simply `ember install` the dependency like normal:
+Now you can simply `ember install` the dependency like normal:
 
 ```sh
 ember install ember-cli-typescript@latest
 ```
 
-##### In addons
-
-To work properly, Ember addons must declare this library as a `dependency`, not a `devDependency`. **This is a *change* from ember-cli-typescript v1.**
-
-1. Remove ember-cli-typescript from your dependencies.
-
-    With yarn:
-    
-    ```sh
-    yarn remove ember-cli-typescript 
-    ```
-
-    With npm:
-    
-    ```sh
-    npm uninstall ember-cli-typescript
-    ```
-
-2. Re-install it with `ember install`:
-
-    ```sh
-    ember install ember-cli-typescript@latest --save
-    ```
+***Note:* To work properly, starting from v2, ember-cli-typescript must be declared as a `dependency`, not a `devDependency` for addons. With `ember install` this migration will be automatically handled for you.**
```

Comment:
open to suggestions to articulate it more accurate

Reply:
I like what you have here. I would love it if we added back in the earlier instructions, perhaps like so:

```markdown
If you choose to make the upgrade manually with Yarn or npm, here are the steps you need to follow:
```

Reply:
I've bring non-`ember install` instructions back, but it still requires `ember generate` to work properly. Please check, and let me know if it makes sense to you.